packages/notifi-react-example/src/App.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 
 import './App.css';
 import { NotifiCard } from './NotifiCard/NotifiCard';
+import './Potem.css';
```

Comment:
does this need to be in here?

Reply:
ah no, but it's a cool demo of what's possible with theming.